src/dependencies.jl

```diff
@@ -106,7 +106,7 @@ end
 can_use(::Type{AptGet}) = has_apt && OS_NAME == :Linux
 package_available(p::AptGet) = can_use(AptGet) && !isempty(available_versions(p))
 function available_versions(p::AptGet)
-    vers = ASCIIString[]
+    vers = String[]
```

Comment:
shall we tag Compat again and use `Compat.ASCIIString` to be on the safe side here then?